core/src/main/java/com/microsoft/applicationinsights/channel/concrete/inprocess/InProcessTelemetryChannel.java

```diff
@@ -181,11 +210,42 @@ public void flush() {
         telemetryBuffer.flush();
     }
 
+    /**
+     * Sets the buffer size
+     * @param maxTelemetriesInBatch should be between MIN_NUMBER_OF_TELEMETRIES_PER_CONTAINER
+     *                              and MAX_NUMBER_OF_TELEMETRIES_PER_CONTAINER inclusive
+     */
+    public void setMaxTelemetriesInBatch(int maxTelemetriesInBatch) {
+        if (maxTelemetriesInBatch >= MIN_NUMBER_OF_TELEMETRIES_PER_CONTAINER &&
```

Comment:
Here you're using **Batch** while in other place you're using "Queue". Please be consistent (I personally prefer "batch").